### ISSUE TITLE
User flow issue

### ISSUE THREAD

**Issue by hi-rafa** [magento/magento2#11311]
I was adding a new attribute to a configuration product and I had to leave for a while. When I came back and I keep on working my user session expired. When I clicked on the "Create New Attribute" button, the modal opened and asked me to login. After I inserted my credentials rather than allowing me to keep on working it showed me the admin home panel.

Magento ver. 2.1.9

### Steps to reproduce

1. Sing in Admin panel
2. Go to catalog product
3. Add a configurable product
4. Create configuration
5. WAIT 10 minutes Or delete user session
6. Create new attribute

### Actual result

1. [Screenshot, logs]
![step1](https://user-images.githubusercontent.com/3813277/31338985-acf231a6-ad01-11e7-9407-e3ee3811afb1.png)
![step2](https://user-images.githubusercontent.com/3813277/31338990-b0cedc3e-ad01-11e7-8c79-57848658c643.png)
![step3](https://user-images.githubusercontent.com/3813277/31338999-b3d2b734-ad01-11e7-80f3-449e8aa5f985.png)
![step4](https://user-images.githubusercontent.com/3813277/31339003-b6c30930-ad01-11e7-9777-673373b23bc8.png)
![step5](https://user-images.githubusercontent.com/3813277/31339006-b92aa476-ad01-11e7-977b-b3855fb0ac4e.png)

**Comment by magento-engcom-team** [comment]
@hi-rafa, thank you for your report.
We've created internal ticket(s) MAGETWO-81316 to track progress on the issue.

**Comment by hi-rafa** [comment]
Would it be ok to redirect the user to the Sign in page?
This behavior should be apply to all the magento2 admin panel sections where this issue occurs right?

**Comment by Silarn** [comment]
This is true any time your session expires in Magento 2. Magento 1 - in may cases though not all - could redirect you back to where your session expired. As far as I can tell, Magento 2 does not do that.
